### ISSUE TITLE
r10k requires version specification for Forge modules in Puppetfile

### ISSUE THREAD

**Issue by antaflos** [puppetlabs/r10k#21]
As far as I understand the Puppetfile format allows specifying Forge modules without a version number. It seems r10k doesn't support that:

```
[R10K::Action::Environment::Deploy - INFO] Deploying environment testing

Runtime error: #<RuntimeError: Module puppetlabs/apt with args [] doesn't have an implementation. (Are you using the right arguments?)>
```

The Puppetfile in question:

```
forge "http://forge.puppetlabs.com"

mod "puppetlabs/apt"
...
```

Adding a version specification makes it work again:

```
forge "http://forge.puppetlabs.com"

mod "puppetlabs/apt", "1.1.0"
```

**Comment by adrienthebo** [comment]
Strange... I could have _sworn_ that I fixed this at some point but it looks like I didn't. Given something like `mod "foo/bar", would you say that the slash is enough to indicate that it's a Forge module? What should the default behavior if a module doesn't have any arguments?

**Comment by antaflos** [comment]
I think the behaviour should be similar to librarian-puppet, shouldn't it? We haven't used librarian-puppet extensively but I believe the slash is enough to indicate a Forge module and without any version arguments it gets the latest available version.

**Comment by adrienthebo** [comment]
I agree with you, and I'll update the detection to behave accordingly.

**Comment by adrienthebo** [comment]
Merged in 9ef872cedf4aba9575588e60cdea9ffe35c3dda3; this should be released in 1.0.0. Thanks again for the report!

**Comment by adrienthebo** [comment]
Released in 1.0.0rc1
